Makefile

```diff
@@ -10,8 +10,9 @@ UNITWORKDIR = "/var/lib/neo-go"
 
 DC_FILE=.docker/docker-compose.yml
 
+GOOS ?= $(shell go env GOOS)
 REPO ?= "$(shell go list -m)"
-VERSION ?= "$(shell git describe --tags 2>/dev/null | sed 's/^v//')"
+VERSION ?= "$(shell git describe --tags 2>/dev/null | sed 's/^v//')$(shell if [ "$(GOOS)" = "windows" ]; then echo "_$(GOOS)_unsupported"; fi)"
```

Comment:
https://github.com/nspcc-dev/neo-go/blob/a0473aca92cdd875a7d5a81b3173d73114c05386/pkg/smartcontract/nef/nef.go#L62-L68

Reply:
So currently it's `neo-go-0.97.4-pre-189-gc3fc5835_windows_unsupported`, i.e. 51 symbols at max, and the maximum len for compiler name is 64. I think it's enough free space. Or should we change it?

Reply:
Also, I'm keeping in mind that we may want to add support for MacOS in future, and this script can be easily extended for it.

Reply:
Looks like I'm not up to date wrt 719dceff7. It fits now, so we're fine. But do we need `_windows` here? It's an exe file, it's obviously for Windows (notice that Docker images do need some difference in tag to separate them from Linux ones).

Reply:
I added `_windows` mostly for external users, i.g. those who will query `getversion` or `getcontractstate` RPC calls. These users are not aware of node's executable file.

But yes, it may be overhead.

Reply:
Fixed.

Reply:
RPC users shouldn't care, it's the same version, just a different build.